package.json

```diff
@@ -29,5 +31,9 @@
   "bugs": {
     "url": "https://github.com/arshadkazmi42/strmat/issues"
   },
-  "homepage": "https://github.com/arshadkazmi42/strmat#readme"
+  "homepage": "https://github.com/arshadkazmi42/strmat#readme",
+  "dependencies": {
```

Comment:
This does not belongs here.
Can you remove the dependencies.
It does not uses any dependencies and same package we cannot use as dependency.
So complete depencies object should be removed